README.md

```diff
@@ -1,3 +1,12 @@
+## KNOT Android library
+
+1. This lib is a part of KNOT solution (for more information: https://github.com/CESARBR) and has aims to provide abstractions for Android application create owner IoT (Internet of Things) solution in KNOT platform by HTTP protocol and via SocketIO.
```

Comment:
Please, fix text. It should be

    ...and it aims to provide an abstraction for Android applications to create IoT (Internet of Things) solutions using KNOT platform by HTTP and Socket protocols.